frontend/src/components/Mobile/InputField.tsx

```diff
@@ -55,43 +61,44 @@ const StyledTitle = styled.div`
 `
 
 type InputFieldProps = {
-  title?: string
+  name: string
+  title: string
   placeholder: string
-  value?: string
-  setValue?: Dispatch<SetStateAction<string>> | ((input: string) => void)
   textArea?: boolean
-  hasError?: boolean
-  onChange?: () => void
+  required?: boolean
+  register
+  setValue
+  errors
```

Comment:
pass in the exact error instead

Reply:
also pls include they types, which can be found in the documentation

Reply:
[eaf1e41](https://github.com/rhdevs/rhapp/pull/637/commits/eaf1e41bb6342df75f61972346b27a3bb78b5846)